recipes/kubernetes-helm/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+make_goroot_read_only()
+{
+    find $GOROOT -type d -exec chmod 555 {} \;
+}
+
+make_gopath()
+{
+   export GOPATH=$(pwd)/go
+   export PATH=$GOPATH/bin:$PATH
+
+   export GOSRC_DIR=$GOPATH/src/k8s.io/helm
+   mkdir -p $GOSRC_DIR
+
+   find . -maxdepth 1 \
+       -not -name '.' -and \
+       -not -name 'go' -and \
+       -not -name 'LICENSE' \
+       -exec mv {} $GOSRC_DIR/ \;
+   cd $GOSRC_DIR
+}
+
+build_unix()
+{
+    make bootstrap
+    make build
+    make test-unit
+}
+
+make_goroot_read_only
+make_gopath
+
+case $(uname -s) in
+    "Linux")
+        build_unix
+        cp bin/{helm,rudder,tiller} $PREFIX/bin
```

Comment:
Can you please remind me why `rudder` and `tiller` don't exist for macOS?

Reply:
`rudder` and `tiller` are not available in the upstream packaged binaries. They are usually used inside a kubernetes pod.

Reply:
I see. Does it make sense to have them as binaries on Linux then or is the intention that the conda package might be installed within the pod in some cases?